android/src/main/java/com/twilio/twilio_voice/AnswerJavaActivity.java

```diff
@@ -397,8 +397,8 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
     @Override
     protected void onDestroy() {
         Log.d(TAG, "AnwserJAvaActivity ondestroy");
-//        unregisterReceiver();
         super.onDestroy();
+        unregisterReceiver();
```

Comment:
This caused a memory leak when not unregistering but did not cause the UI to remain after finishing.